tokens/eth/0x8d5682941ce456900b12d47ac06a88b47c764ce1.json

```diff
@@ -0,0 +1,32 @@
+{
+    "symbol": "RMESH",
+    "address": "0x8d5682941ce456900b12d47ac06a88b47c764ce1",
+    "decimals": 18,
+    "name": "RightMesh Token",
+    "ens_address": "",
+    "website": "https://www.rightmesh.io/",
+    "logo": {
+      "src": "https://avatars2.githubusercontent.com/u/29805586?s=128",
+      "width": "92",
+      "height": "92",
+      "ipfs_hash": "QmQ9tzSaUpjrhsZ6vFjp8P5FcBWFEKrYD8joi6d8HVXcdc"
```

Comment:
Dunno if im just dumb at ipfs or what - but I can't seem to get this hash to resolve.

Reply:
Sorry about that, I forgot to add it to my server (my local computer wasn't connected when you checked it). I've now put it on a server that is reliable and is supposed to work 24/7 but I would recommend having at least 1 IPFS node that hosts the same file on your end (doing a simple `ipfs get  QmQ9tzSaUpjrhsZ6vFjp8P5FcBWFEKrYD8joi6d8HVXcdc` will make it so your own server also hosts it and adds it to the DHT

Reply:
Perfect - yep have an ipfs node running permanently on my miner at home, so I'll do the get there. Tried on my local machine and have it now as well - looks good.